frontend/src/components/VResult.vue

```diff
@@ -7,58 +7,36 @@
       </div>
     </div>
 
-    <h4 class="mt-3 mb-2">Resultater</h4>
-    <div class="card mt-2">
+    <h4 v-if="items.length" class="mt-3 mb-2">{{ $t('results') }}</h4>
+    <h4 v-if="!items.length" class="mt-3 mb-2">{{ $t('no_results') }}</h4>
+
+    <div class="card mt-2 mb-2" v-for="item in items" :key="item.uuid" v-if="!item.parent">
```

Comment:
here the key is fine, because it's clear what the value is supposed to be :) (`v-if` is still an issue though)

Reply:
Still,`v-if` is an issue here